aws-py-apigatewayv2-eventbridge/__main__.py

```diff
@@ -0,0 +1,132 @@
+# Copyright 2016-2021, Pulumi Corporation.  All rights reserved.
```

Comment:
There's some [PEP8](https://www.python.org/dev/peps/pep-0008/) whitespace stuff (blank line counts, indentation) that I'd change here. They're not crucial/blockers as the code certainly will still run, but I'd recommend following them if we can as some Python devs might give this a bit of side-eye. Would you like me to fix them?

Reply:
Oh sorry, yes -- definitely feel free to adjust anything you think needs adjusting space-wise. Thank you!